using_images/xpaas_images/decision_server.adoc

```diff
@@ -0,0 +1,137 @@
+= Decision Server xPaaS Image
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+Decision Server is available as a containerized xPaaS image that is designed for use with OpenShift as an execution environment for business rules.  Developers can quickly build, scale, and test applications deployed across hybrid environments.
+
+[IMPORTANT]
+There are significant differences in supported configurations and functionality
+in the Decision Server xPaaS image compared to the regular release of JBoss BRMS.
+
+This topic details the differences between the Decision Server xPaaS image and the
+regular release of JBoss BRMS, and provides instructions specific to running and
+configuring the Decision Server xPaaS image. Documentation for other JBoss BRMS
+functionality not specific to the Decision Server xPaaS image can be found in the
+https://access.redhat.com/documentation/en/red-hat-jboss-brms/[JBoss
+BRMS documentation on the Red Hat Customer Portal].
+
+`_EAP_HOME_` in this documentation, as in the
+https://access.redhat.com/documentation/en/red-hat-jboss-brms/[JBoss
+BRMS documentation], is used to refer to the JBoss EAP installation directory 
+where the decision server is deployed. The location of `_EAP_HOME_` inside a 
+Decision Server xPaaS image is *_/opt/eap/_*, which the `*JBOSS_HOME*` 
+environment variable is also set to by default.
+
+== Comparing the Decision Server xPaaS Image to the Regular Release of JBoss BRMS
+
+=== Functionality Differences for OpenShift Decision Server xPaaS Images
+
+There are several major functionality differences in the OpenShift Decision Server xPaaS image:
+
+* The Decision Server image extends the Openshift EAP image, and any capabilities or limitations it has are also found in the Decision Server image.
+* Only stateless scenarios are supported.
+* Execution of business processes are not supported.
+* Authoring of business rules are not supported.
+
+[[Managing-OpenShift-Decision-Server-xPaaS-Images]]
+=== Managing OpenShift Decision Server xPaaS Images
+
+As the Decision Server image is built off the OpenShift JBoss EAP xPaaS image the JBoss EAP Management CLI
```

Comment:
@huffmanca A little nit-picky, but there should be a comma after xPaas image. So:

"As the Decision Server image is built off the OpenShift JBoss EAP xPaaS image, the JBoss EAP..."